spot_driver/src/spot_driver/spot_wrapper.py

```diff
@@ -674,6 +695,238 @@ def navigate_to(self, upload_path,
 
         return resp
 
+    # Arm ############################################
+    def check_arm_power_and_stand(self):
+        if not self._robot.has_arm():
+            return False, "Spot with an arm is required for this service"
+        
+        try:
+            self._logger.info("Spot is powering on within the timeout of 20 secs")
+            self._robot.power_on(timeout_sec=20)
+            assert self._robot.is_powered_on(), "Spot failed to power on"
+            self._logger.info("Spot is powered on")
+            robot_command.blocking_stand(command_client=self._robot_command_client, timeout_sec=10.0)
+            self._logger.info("Spot is standing")
+
+        except Exception as e:
+            return False, "Exception occured while Spot was trying to power on or stand"
+
+        return True, "Spot has an arm, is powered on, and standing"
+
+    
+    def arm_stow(self):
+        try:
+            self.check_arm_power_and_stand()
```

Comment:
```suggestion
            if not self.check_arm_power_and_stand():
                  return
```

Since check_arm_power_and_stand returns a boolean, we also have to exit the method here, if there is no arm.
Same with all other calls to this function in all Arm services.